go/vt/vtgate/executor_select_test.go

```diff
@@ -509,7 +509,7 @@ func TestLastInsertIDInSubQueryExpression(t *testing.T) {
 	_, err := executorExec(executor, "select (select last_insert_id()) as x", nil)
 	require.NoError(t, err)
 	wantQueries := []*querypb.BoundQuery{{
-		Sql:           "select (select :__lastInsertId as `last_insert_id()` from dual) as x from dual",
+		Sql:           "select last_insert_id() as x from dual",
 		BindVariables: map[string]*querypb.BindVariable{"__lastInsertId": sqltypes.Uint64BindVariable(0)},
 	}}
```

Comment:
expected sql query looks wrong to me
this should be `select :__lastInsertId as x from dual`